public/index.html

```diff
@@ -40,7 +40,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
         integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
-    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,400i" rel="stylesheet">
```

Comment:
In my opinion it would be better for performance to do :
`<link rel="preload" href="https://fonts.googleapis.com/css?family=Lato:400,400i" as="font" type="font/woff2" crossOrigin="true">`

Reply:
@tureey What's the difference? I am gonna blindly add it right away, and gonna do a check on `type` and `as`. I don't think they are required really.

Reply:
here is explained https://medium.com/clio-calliope/making-google-fonts-faster-aadf3c02a36d

Reply:
Thanks man... 😇